### ISSUE TITLE
Deprecate `listaddressgroupings`

### ISSUE THREAD

**Issue by nuttycom** [zcash/zcash#5940]
This RPC api is leftover privacy theater from Bitcoin. We should remove it.

**Comment by benjamin203** [comment]
I agree. Its purpose was to help users to try to keep their funds in different address groups that couldn't be linked together. If someone wanted to do this with zcashd, they should create multiple accounts and keep their funds within each account's addresses.
